src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BlobContainerClientProxy.java

```diff
@@ -14,22 +15,30 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.blobrouter.config.TargetStorageAccount;
+import uk.gov.hmcts.reform.blobrouter.exceptions.InvalidZipArchiveException;
 
 import java.io.ByteArrayInputStream;
+import java.io.IOException;
 import java.time.Duration;
 import java.time.LocalDateTime;
 import java.time.OffsetDateTime;
 import java.time.ZoneOffset;
 import java.time.temporal.ChronoUnit;
 import java.util.Map;
+import java.util.Objects;
 import java.util.concurrent.TimeUnit;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
 
 import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.hmcts.reform.blobrouter.util.zipverification.ZipVerifiers.ENVELOPE;
 
 @Service
 public class BlobContainerClientProxy {
 
     private static final Logger logger = getLogger(BlobContainerClientProxy.class);
+    public static final int BUFFER_SIZE = 10240;
+    public static final long BLOCK_SIZE = 1024L * 50L;
```

Comment:
Can you explain this?

Reply:
it is the max request size. if you make it too big it consumes memory, if you make it too small it increase the request number.

Reply:
I mean - define both figures as constants with self-explanatory names.

Reply:
I think multiplication is fine , easy to understand. I can change the names , do you have any suggestion ?

Reply:
I do not mind the multiplication, but what is 1024L and what is 50L?

Reply:
added comment